### ISSUE TITLE
Help maintaing

### ISSUE THREAD

**Issue by deeg** [angular-ui/ui-tinymce#312]
Hey everyone, I'm sorry about the limited time I have had to maintain this library.

Things at work have been crazy and have been focusing on things Angular 2.

If you would like to help maintain the library please contact me either in this post or via email so we can talk.

**Comment by deeg** [comment]
Bumping this to the top. Still looking for help maintaining the library.

**Comment by Jbz797** [comment]
Have you started working on the Angular 2 version?

**Comment by deeg** [comment]
No, [TinyMCE has documented](https://www.tinymce.com/docs/integrations/angular2/) how to use an editor in Angular 2 though.

I am using Angular 2 at work now, and when I need an editor I was going to write a wrapper if one does not yet exist if it seems like it is needed.

**Comment by ChristianKreuzberger** [comment]
Hi there,

I would like to help maintaining this project. Please let me know how to proceed, and if you need any details.

Best regards
Christian Kreuzberger
